### PR TITLE
Implemented a twist_to_jogframe controller

### PR DESCRIPTION
**Pull request by LazyEngineerToBe**
This PR has had a code review on our end.

### CODE REVIEW

**Comment by LazyEngineerToBe** [comment]
I didn't write a test for the new controller. Hence why the build fails.

**Comment by 7675t** [comment]
Thanks for your contribution.
I’ｍ checking them in a few days.

File: jog_controller/launch/twist.launch [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<launch>
+  <arg name="joy_config" default="spacemouse_twist"/>
+  <arg name="group_name" default="manipulator"/>
+  <arg name="link_name" default="tool0"/>
+  <arg name="frame_id" default="tool0"/>
+  <arg name="sub_topic" default="spacenav/twist"/>
+
+  <include file="$(find spacenav_node)/launch/classic.launch"/>
```
Comment (7675t) [inline jog_controller/launch/twist.launch]: This makes CI fails. We should add dependency on spacenav_node in package.xml

**Comment by 7675t** [comment]
Thanks for your contribution. I understand this PR for SpaceNav (Unfortunately I don't have it).

Are these are correct?
- SpaceNav produce joy AND twist message
- Current joy_to_jog_frame.py frame is not enough to control the robot with SpaceNav
- So you add twist_to_jog_frame.py to control your robot with SpaceNav
- You have confirmed you can control your robot with it, and you are totally happy :smile: 

I have no problem to merge this. Please fix the dependency to pass the CI.

File: jog_controller/launch/twist.launch (hunk 1 above)
Comment (LazyEngineerToBe) [reply jog_controller/launch/twist.launch]: I couldn't figure out what was wrong, thanks! I'll fix it.

**Comment by LazyEngineerToBe** [comment]
> Thanks for your contribution. I understand this PR for SpaceNav (Unfortunately I don't have it).
> 
> Are these are correct?
> 
> * SpaceNav produce joy AND twist message
> * Current joy_to_jog_frame.py frame is not enough to control the robot with SpaceNav
> * So you add twist_to_jog_frame.py to control your robot with SpaceNav
> * You have confirmed you can control your robot with it, and you are totally happy 😄
> 
> I have no problem to merge this. Please fix the dependency to pass the CI.

Yes to all of the above. Tested it using the simulated motoman_sia20d robot.

**Comment by LazyEngineerToBe** [comment]
@7675t Weird, that travis-ci couldn't compile this PR as I'm running on ubuntu 18.04

**Comment by 7675t** [comment]
@LazyEngineerToBe Don't worry about it fails in melodic. It needs some package released.

**Pull request merged**